File: python/Elektronik.py
class Elektronik:
    # Constructor
    def __init__(self, id="", nama="", merek="", deskripsi="", harga=""):
        self.id = id
        self.nama = nama
        self.merek = merek
        self.deskripsi = deskripsi
        self.harga = harga

    # Getter and setter for id
    def get_id(self):
        return self.id

    def set_id(self, id):
        self.id = id

    # Getter and setter for nama
    def get_nama(self): 
        return self.nama
    
    def set_nama(self, nama):
        self.nama = nama

    # Getter and setter for merek
    def get_merek(self):
        return self.merek
    
    def set_merek(self, merek):
        self.merek = merek

    # Getter and setter for deskripsi
    def get_deskripsi(self):    
        return self.deskripsi
    
    def set_deskripsi(self, deskripsi):
        self.deskripsi = deskripsi

    # Getter and setter for harga
    def get_harga(self):
        return self.harga
    
    def set_harga(self, harga):
        self.harga = harga

File: php/Elektronik.php
<?php
// Define the Elektronik class
class Elektronik {
    // Private attributes 
    private $id, $nama, $merek, $deskripsi, $harga, $foto;

    // Constructor
    public function __construct($id="", $nama="", $merek="", $deskripsi="", $harga="") {
        $this->id = $id;
        $this->nama = $nama;
        $this->merek = $merek;
        $this->deskripsi = $deskripsi;
        $this->harga = $harga;
        $this->foto = $foto;
    }

    // Getter and setter for id
    public function getId() { return $this->id; }
    public function setId($id) { $this->id = $id; }

    // Getter and setter for nama
    public function getNama() { return $this->nama; }
    public function setNama($nama) { $this->nama = $nama; }

    // Getter and setter for merek
    public function getMerek() { return $this->merek; }
    public function setMerek($merek) { $this->merek = $merek; }

    // Getter and setter for deskripsi
    public function getDeskripsi() { return $this->deskripsi; }
    public function setDeskripsi($deskripsi) { $this->deskripsi = $deskripsi; }

    // Getter and setter for harga
    public function getHarga() { return $this->harga; }
    public function setHarga($harga) { $this->harga = $harga; }
}